issuing_using_did.md

```diff
@@ -0,0 +1,224 @@
+###### Authors:
+
+Raymond Yeh
+
+# Identify Issuer & Issuing Document using DID
+
+## Goal
+
+As a new issuer, I would like to issue verifiable documents without using ethers to reduce cost, but yet being able to be identified via DNS or DID.
+As an existing issuer of verifiable document and transferable records, I would like to have my identity bound to a DID instead of DNS only.
+
+The problems describe above have two facet:
+
+1. How might we identify issuers of document via DID, especially if they are using issuing transferable records?
+2. How might an issuer who is identified with DID be able to issue verifiable document directly by signing on it?
+3. How might we revoke a document issued directly with DID?
+
+## Background
+
+There were request to the team for issuers to be able to sign on a document directly using an Ethereum private key directly. The original solution was include a new proving method to allow issuers to use a key to sign on the document [[1]](https://github.com/Open-Attestation/oa-verify/pull/91). There were suggestions as part of the original discussion which propose how might one be able to associate the unknown key to a DNS by inserting a DNS record to bind the relationship [[2]](https://github.com/Open-Attestation/adr/pull/22).
+
+## Approach
+
+This document revisits the entire discussion to form an understanding of the guiding principles which leads to implementation detail at the end.
+
+## Guiding Principles
+
+1. The solution should be guided by W3C VC standards as far as possible
+2. The solution should allow a less expensive way to issue verifiable document
+3. The solution should allow revocation of said document
+4. The solution should extend to DID
+5. The solution should leverage on the scale of DID for maximum reach
+
+## Overview
+
+![Overview](./assets/issuing_using_did/overview.png)
+
+The proposed solution is to allow DID's to be used a terminal identifier in OA viewers.
+
+![Extending terminal identifiers](./assets/issuing_using_did/terminal_identifiers.png)
+
+The approach will extend the current approach of using only DNS as the terminal identifier and also allow for DID's. While certain identifiers such as ethereum addresses may not be human-friendly, they are still useful for machines which are able to process them. This allows anyone with DID to be able to associate it with their issued documents.
+
+Optionally, one may also associate their DID with a domain name to allow both the DID and the domain name to be recognized as the issuer.
```

Comment:
How ?

Reply:
That will be via dns-txt, where the the entry is something like:

```
openatts did:ethr:0xabcd1234
```